### PR TITLE
fix: use `pathe`

### PR DESCRIPTION
**Pull request by sudongyuer**

### CODE REVIEW

File: packages/vscode/package.json [hunk 1]
```diff
@@ -69,6 +69,9 @@
     "dev": "tsup --watch src",
     "publish": "esno ./scripts/publish.ts"
   },
+  "dependencies": {
+    "pathe": "^0.3.7"
```
Comment (antfu) [inline packages/vscode/package.json:73]: This should go devDeps as VS Code is not a npm package.

**Comment by antfu** [comment]
Is there any real issue to resolve?

File: packages/vscode/package.json (hunk 1 above)
Comment (sudongyuer) [reply packages/vscode/package.json:73]: resolved

**Comment by sudongyuer** [comment]
> Is there any real issue to be resolved by this?
When i write some tests on windows to contribute, be possible to occur. 
image

**Review by antfu — APPROVED** [review]

**Pull request merged**